pkg/serverless/trace/trace.go

```diff
@@ -131,9 +147,44 @@ func filterSpanFromLambdaLibraryOrRuntime(span *pb.Span) bool {
 			return true
 		}
 	}
+
+	if val, ok := span.Meta[tcpRemoteHostMetaKey]; ok {
+
+		if val1, ok := span.Meta[tcpRemotePortMetaKey]; ok {
```

Comment:
We typically want to avoid reusing these generic variable names if possible like `val1`, `val2`, `val3`. Why not `tcpHost` and `tcpPort`? Better yet, for each block (HTTP, TCP, DNS), let's rename the `val` variables as to not reuse them. E.g.: `httpUrl`, `dnsHost`, etc